package.json

```diff
@@ -31,15 +31,15 @@
   "dependencies": {
     "@bsull/augurs": "^0.9.0",
     "@emotion/css": "11.13.5",
-    "@grafana/data": "^12.1.0",
+    "@grafana/data": "12.2.0-17055924271",
     "@grafana/faro-web-sdk": "^1.17.2",
-    "@grafana/i18n": "^12.1.0-249189",
+    "@grafana/i18n": "12.2.0-17055924271",
     "@grafana/lezer-logql": "^0.2.7",
     "@grafana/prometheus": "^12.1.0",
-    "@grafana/runtime": "^12.1.0",
-    "@grafana/scenes": "^6.27.2",
-    "@grafana/schema": "^12.1.0",
-    "@grafana/ui": "^12.1.0",
+    "@grafana/runtime": "12.2.0-17055924271",
+    "@grafana/scenes": "^6.29.7",
+    "@grafana/schema": "12.2.0-17055924271",
+    "@grafana/ui": "12.2.0-17055924271",
```

Comment:
We need to wait for the actual `v12.2.0` to release  before actually being able to release this PR